app/src/main/res/mipmap-anydpi-v26/ic_app_icon.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_app_icon_background"/>
+    <foreground android:drawable="@drawable/ic_app_icon_foreground"/>
+</adaptive-icon>
```

Comment:
I notice that github always warns about having a last empty line at the end of every file. Lint also complains about that. Should we start checking that?